src/angulartics-azure.js

```diff
@@ -0,0 +1,46 @@
+﻿/**
+ * @license Angulartics v0.17.2
+ * (c) 2013 Luis Farzati http://luisfarzati.github.io/angulartics
+ * Microsoft Azure Application Insights plugin contributed by https://github.com/anthonychu
+ * License: MIT
+ */
+(function (angular) {
+    'use strict';
+
+    /**
+     * @ngdoc overview
+     * @name angulartics.azure
+     * Enables analytics support for Microsoft Azure Application Insights (http://azure.microsoft.com/en-us/services/application-insights/)
+     */
+    angular.module('angulartics.azure', ['angulartics'])
+    .config(['$analyticsProvider', function ($analyticsProvider) {
+
+        $analyticsProvider.registerPageTrack(function (path) {
+            appInsights.trackPageView(path);
+        });
+
+        /**
+         * Numeric properties are sent as metric (measurement) properties.
+         * Everything else is sent as normal properties.
+         */

```

Comment:
I haven't used this provider before, but isn't it too much to assume all numeric properties are always metrics? Isn't there a case where I would like to track just a normal numeric property?